### PR TITLE
First draft for Landscape: "SSL certificate" term entry #14

### PR DESCRIPTION
**Pull request by Owslla**
Providing first draft of the ssl-cert.md in path: `landscape/explanation/glossary/ssl-cert.md`

### CODE REVIEW

**Comment by Owslla** [comment]
Curious if the second paragraph is providing too much detail or not closely bound enough to the SSL-cert explanation.

File: landscape/explanation/glossary/ssl-cert.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# SSL Certificates
+
+**Secure Sockets Layer (SSL) certificates** are part of creating secure communication over the internet. They provide a process for encrypting and authenticating data sent between a client and a server. In the context of Landscape, being a server management tool, it uses SSL certificates to communicate securely with its servers, reducing the chance of unauthorized access and data breaches.
```
Comment (YanisaHS) [inline landscape/explanation/glossary/ssl-cert.md:3]:
```suggestion
**Secure Sockets Layer (SSL) certificates** are part of creating secure communication over the internet. They provide a process for encrypting and authenticating data sent between a client and a server. In the context of Landscape, being a systems management tool, it uses SSL certificates to communicate securely with its servers, reducing the chance of unauthorized access and data breaches.
```

Comment (YanisaHS) [reply landscape/explanation/glossary/ssl-cert.md:3]: "Server" is really _wrong_, but "systems management" just feels a bit more precise and it is usually how I refer to it in the documentation

Comment (Owslla) [reply landscape/explanation/glossary/ssl-cert.md:3]: Thank you for the comment.  Can you walk me through this part of the review process? Should I go back and edit the original file on the pull request, or use the “Commit suggestion” option below your suggested changes?

File: landscape/explanation/glossary/ssl-cert.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+If you install Landscape with a public IP, the [Quickstart installation](https://ubuntu.com/landscape/docs/quickstart-deployment) guide recommends you install an SSL certificate from a certificate authority, such as LetsEncrypt. The SSL certificate is used to secure the communication between the Landscape server and the clients (the servers being managed) as the data travels over public networks. This ensures that only the intended recipient can decrypt and access it.  In other words, if someone were able to intercept the communication, they would not be able to understand the data being sent because it is encrypted.
```
[the remark this answers is not in the document]
Comment (YanisaHS) [reply landscape/explanation/glossary/ssl-cert.md:5]: The clients in Landscape aren't always servers - they can be other devices

File: landscape/explanation/glossary/ssl-cert.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+> Think of Landscape as a post office and SSL certificates as special keys to mailboxes. Each managed server is like a mailbox. To send a secure message, the post office (Landscape) locks the message (data) with a special key (SSL certificate). Only the mailbox (managed server) with the matching key can open it. This is how Landscape communicates securely with servers using an SSL certificate.
```
Comment (YanisaHS) [inline landscape/explanation/glossary/ssl-cert.md:7]:
```suggestion
You can think of Landscape as a post office system and SSL certificates as special keys to specific mailboxes. The mailboxes represent the network connections between Landscape servers and clients. The Landscape server is like a mail carrier, and the Landscape clients are the recipients. To send a secure message, the mail carrier (the Landscape server) locks the message (data) with a special key (SSL certificate). Only the recipient (the Landscape client) with the matching key can open it. This is how Landscape communicates securely with servers using an SSL certificate.
```

Comment (YanisaHS) [reply landscape/explanation/glossary/ssl-cert.md:7]: Just making it a bit more clear :smiley: Great example though!

**Comment by YanisaHS** [comment]
@Owslla Thanks so much for sending the PR! You did a great job! I'm done reviewing it. Please let me know when you're done reviewing it too and are ready for me to take another look.

And to answer your previous question about the review process: You can either edit the file locally and push your commits back into the PR, or you can just click **Commit suggestion** and they will be committed/resolved in GH. (Or leave a comment and tag me if you want clarification or have a question/concern before you commit)

Personally, when I'm working with a reviewer like you are, I like to commit the suggestions in GH first. Then, if I want to make additional changes, I'll pull the new commits down locally, edit my files locally, and push everything back into the PR. Since I didn't leave any ambiguous suggestions for you, you could just commit each suggestion if you're okay with them. :smiley:

File: landscape/explanation/glossary/ssl-cert.md (hunk 1 above)
Comment (YanisaHS) [reply landscape/explanation/glossary/ssl-cert.md:3]: @Owslla Just answered this in a comment below :smile:

**Pull request merged**